My suggestion was missing a `let`:

```
error[E0425]: cannot find value `family_name` in this scope
  --> components\fonts\platform\windows\font_list.rs:28:15
   |
28 |         if Ok(family_name) = family.family_name() {
   |               ^^^^^^^^^^^ not found in this scope
   |
help: you might have meant to use pattern matching
   |
28 |         if let Ok(family_name) = family.family_name() {
   |            +++

error[E0425]: cannot find value `family_name` in this scope
  --> components\fonts\platform\windows\font_list.rs:29:22
   |
29 |             callback(family_name);
   |                      ^^^^^^^^^^^ not found in this scope

error[E0308]: mismatched types
  --> components\fonts\platform\windows\font_list.rs:28:12
   |
28 |         if Ok(family_name) = family.family_name() {
   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `bool`, found `()`
   |
help: consider adding `let`
   |
28 |         if let Ok(family_name) = family.family_name() {
   |            +++
```

--- a/components/fonts/platform/windows/font_list.rs
+++ b/components/fonts/platform/windows/font_list.rs
@@ -25,7 +25,9 @@ where
 {
     let system_fc = FontCollection::system();
     for family in system_fc.families_iter() {
-        callback(family.family_name().unwrap());
+        if let Ok(family_name) = family.family_name() {
+            callback(family_name);
+        }
     }
 }
 
@@ -95,7 +97,9 @@ where
     if let Ok(Some(family)) = system_fc.font_family_by_name(family_name) {
         let count = family.get_font_count();
         for i in 0..count {
-            let font = family.font(i).unwrap();
+            let Ok(font) = family.font(i) else {
+                continue;
+            };
             let template_descriptor = (&font).into();
             let local_font_identifier = LocalFontIdentifier {
                 font_descriptor: Arc::new(font.to_descriptor()),

